### ISSUE TITLE
Colab documentation

### ISSUE THREAD

**Issue by GitHubGeniusOverlord** [marcoppasini/musika#26]
Hello,
I had some problems with the colab.
1. You do not specify the file format allowed. Is it wav? It would be great if you could add this information.
All other problems may come from there. 

Thanks for the great project!

**Comment by marcoppasini** [comment]
Hello!
Which format are your files in? I use ffmpeg to read the audio files so a large variety of formats should be covered.

**Comment by GitHubGeniusOverlord** [comment]
Hi, 
I used flac. Now I tried with wav. Still problematic. Probably the format is not the problem then. 
I am trying to load from google drive and save the encoded files there. 
To achieve this I am putting files_path: "/content/drive/folder_with_wavs" and
save_path: "/content/drive/audio_encodings_folder"

I get this output:
`
Using GPU with mixed precision enabled...

Checking if models are already available...
Models are available!
/usr/local/lib/python3.9/site-packages/keras/initializers/initializers_v2.py:120: UserWarning: The initializer HeUniform is unseeded and being called multiple times, which will return identical values  each time (even if the initializer is unseeded). Please update your code to provide a seed to the initializer, or avoid using the same initalizer instance more than once.
  warnings.warn(
Encoders/Decoders loaded from checkpoints/ae
Networks initialized
0it [00:00, ?it/s]
`
So basically nothing happens there.

**Comment by marcoppasini** [comment]
Ok, got it!
The path to the folder on your google drive is "/content/drive/**MyDrive**/folder_with_wavs" (and consequently "/content/drive/**MyDrive**/audio_encodings_folder").
Right now you are referencing non-existing folders!

You can always check the position of your folder by navigating to it in the "File" tab in colab (the little folder icon on the left). Hope this helps!

**Comment by GitHubGeniusOverlord** [comment]
Thank You @marcoppasini 
I did that. Still did not work, but this was due to my browser. Google Colab and Drive do not like Firefox. After switching to the edge browser it works so far. 
Maybe that is just something to add to the documentation, if you want it to be really user-friendly!
